pages/api/bib/[id]/items.ts

```diff
@@ -18,6 +18,7 @@ async function handler(req: NextApiRequest, res: NextApiResponse) {
       })
     } else {
       res.status(200).json({
+        discoveryBibResult,
```

Comment:
That makes sense. Would it be preferable to pass showItemTable instead of the whole bib? Or would it be better to remove the items endpoint entirely and move the looped items fetch to the bib endpoint and refresh the whole bib?

The first option would be easy to implement on top of what I have, but the second would reduce the amount of code we'll have to maintain. What do you and @charmingduchess think?